### DISCUSSION TITLE
How to customize/template the DataGrid DataGridAggregates Line Display

### DISCUSSION THREAD

**Discussion by sylvain-groulx** [Megabit/Blazorise#4703] [Q&A] [answered]
My PO wants this (aggregate - TOTAL: $2,202.66):

![image](https://user-images.githubusercontent.com/129307827/228588410-b531c0f5-9e64-430e-b20b-61e10fd4bcbf.png)

And I have this (How may I remove the borders on the aggregate line please?): Thank you in advance!

![image](https://user-images.githubusercontent.com/129307827/228589002-85c930cc-32e2-475e-9705-0915ac6f3b93.png)

**Comment by David-Moreira** [comment]
Hello @sylvain-groulx 
I believe you can use the **AggregateCellClass** or **AggregateCellStyle** to apply specific styling that removes them. Have you tried that?

**Comment by David-Moreira** [comment, marked as answer]
Oh, and there's also a **DataGrid** parameter, **GroupRowStyling** that applies styling to the aggregate rows. 
Let us know if you are able to achieve the look you're looking for with these tools.

**Reply by sylvain-groulx** [reply]
Wonderful! 
Thank you

![image](https://user-images.githubusercontent.com/129307827/228672979-ff423668-ccb7-4d28-ad03-cb8c39f900a7.png)

<DataGrid TItem="...
GroupRowStyling="@_transactionState.DataGridState.AggregateRowStyling"

::deep .aggregate-row td:nth-child(-n+4) {
    border-right-style: hidden;
    text-align: right !important;
}

**Reply by David-Moreira** [reply]
Great, thanks for letting us know.
@stsrki I think we've talked briefly before, but we should rename `GroupRowStyling` Parameter to `AggregateRowStyling`, no? Even I got confused with the naming when I was taking a look at the available api.

**Reply by stsrki** [reply]
We have already renamed most of the parameters recently.

```cs
/// <summary>
/// Custom classname for the aggregate cell.
/// </summary>
[Obsolete( "DataGridColumn: The GroupCellClass parameter is deprecated, please use the AggregateCellClass parameter instead." )]
[Parameter] public string GroupCellClass { get => AggregateCellClass; set => AggregateCellClass = value; }

/// <summary>
/// Custom style for the aggregate cell.
/// </summary>
[Obsolete( "DataGridColumn: The GroupCellStyle parameter is deprecated, please use the AggregateCellStyle parameter instead." )]
[Parameter] public string GroupCellStyle { get => AggregateCellStyle; set => AggregateCellStyle = value; }

/// <summary>
/// Custom classname for the aggregate cell.
/// </summary>
[Parameter] public string AggregateCellClass { get; set; }

/// <summary>
/// Custom style for the aggregate cell.
/// </summary>
[Parameter] public string AggregateCellStyle { get; set; }
```

We seem to forget about the `GroupRowStyling`. Shouldn't be too hard to make in 1.3.
